frontend/src/app/shared/components/grids/widgets/time-entries/current-user/time-entries-current-user.component.ts

```diff
@@ -40,7 +40,7 @@ export class WidgetTimeEntriesCurrentUserComponent extends AbstractWidgetCompone
     const duration = this.entries.reduce((current, entry) => current + this.timezone.toHours(entry.hours), 0);
 
     if (duration > 0) {
-      return this.i18n.t('js.units.hour', { count: this.formatNumber(duration) });
```

Comment:
We could use I18n's `numberToRounded` with a 2 digit precision to achieve the same as we had before.

Reply:
Yes we could do that, this is merely a typescript error fix that is now apparent (we passed in a string before, which didn't correctly match the count so it was just returned).